conf/app.routes

```diff
@@ -9,4 +9,5 @@ POST       /undertaking/member/:undertakingRef              uk.gov.hmrc.eusubsid
 POST       /undertaking/member/remove/:undertakingRef       uk.gov.hmrc.eusubsidycompliance.controllers.UndertakingController.deleteMember(undertakingRef)
 
 POST       /subsidy/update                                  uk.gov.hmrc.eusubsidycompliance.controllers.UndertakingController.updateSubsidy()
+POST       /subsidy/remove                                  uk.gov.hmrc.eusubsidycompliance.controllers.UndertakingController.updateSubsidy()
```

Comment:
Shouldn't we call method  as removeSubsidy in here  like uk.gov.hmrc.eusubsidycompliance.controllers.UndertakingController.removeSubsidy()?

Reply:
Nahh the update operation is the same. Just depends on the amendmenttype passed up
1 - add
2 - update
3 - remove

I've removed it to get rid of the ambiguity.